qa/integration/specs/pipeline_log_spec.rb

```diff
@@ -44,4 +44,19 @@
     expect(File.exists?(plainlog_file)).to be true
     expect(IO.read(plainlog_file) =~ /\[logstash.javapipeline\s*\]\[#{pipeline_name}\]/).to be > 0
   end
+
+  it "write pipeline config in logs" do
+    pipeline_name = "custom_pipeline"
+    settings = {
+      "path.logs" => temp_dir,
+      "pipeline.id" => pipeline_name
+    }
+    IO.write(@ls.application_settings_file, settings.to_yaml)
+    @ls.spawn_logstash("-w", "1" , "-e", config)
+    @ls.wait_for_logstash
+    sleep 2 until @ls.exited?
+    plainlog_file = "#{temp_dir}/logstash-plain.log"
+    expect(File.exists?(plainlog_file)).to be true
+    expect(IO.read(plainlog_file) =~ /Starting pipeline.*"pipeline.sources"=>\["config string"\]/).to be > 0
+  end
```

Comment:
Can we also add a test for logstash starting via a config file as well as config string? I think that is probably where this functionality has the most utility.